src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -131,6 +145,13 @@ function checkTheRoleThatApplies(instrNum) {
 	var instrRole = $("#accessControlInfoForInstr" + instrNum + " div div p span").html();
 	$("input[id='instructorroleforinstructor" + instrNum + "']").filter("[value='" + instrRole + "']").prop("checked", true);
 	if (instrRole === "Custom") {
+		// Save original values of Custom Role
+		instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum] = [];
+		for (var i = 0; i < instructorPrivilegeValues.length; i++) {
+			var checkValue = $("#tunePermissionsDivForInstructor" + instrNum + " input[name='" + instructorPrivilegeValues[i] + "']").prop("checked");
+			instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum][instructorPrivilegeValues[i]] = checkValue;
```

Comment:
So informative. Thanks so much. I think my javascript/jQuery skill boosted a lot in this summer xD